src/ui/tabview/content/WebContent.qml

```diff
@@ -23,6 +23,8 @@
 
 import QtQuick 2.0
 import QtQuick.Controls 2.0
+import QtQuick.Dialogs 1.1
+import Fluid.Controls 1.0
```

Comment:
`import Fluid.Controls 1.0 as FluidControls` allows to identify the import in case of conflict with QQC2